<launch>
        <arg name="rosBag" default = "/home/hri/Documents/SLAM_Dataset/RealSense/rosbag_file/room_10000.bag" />
        <arg name="config_file" default = "$(find gyro_predict_matcher)/config/RealSenseD435i.yaml" />

        <!-- set timedelay from bash input -->
        <arg name="manuallyAddTimeDelay" default = "0.000" />
        <!--<arg name="output_file" default = "/../output/timedelay_effect/TD_$(arg manuallyAddTimeDelay)/" />-->
        <arg name="output_file" default = "/../output/" />

        <node pkg="gyro_predict_matcher" type="gyro_predict_and_feature_detect_node" name="gyro_predict_and_feature_detect_node"  args="$(arg config_file)" cwd="node" output="screen">
            <param name="rosBag" type="string" value="$(arg rosBag)" />
            <param name="manuallyAddTimeDelay" type="double" value="$(arg manuallyAddTimeDelay)" />
            <param name="output_file" type="string" value="$(arg output_file)" />

        </node>

</launch>

